markdown/dev/reference/plugins/gore/en.md

```diff
@@ -2,14 +2,14 @@
 title: "@freesewing/plugin-gore"
 ---
 
-The **@freesewing/plugin-gore** plugin provides 
+The **@freesewing/plugin-gore** plugin provides
 [the gore macro](/reference/api/macros/gore).
-This macro allows you to generate [gore 
-segments](https://en.wikipedia.org/wiki/Gore_(segment)) — 
-2D panels to create a sphehrical shape as used in hats for example — 
+This macro allows you to generate [gore
+segments](https://en.wikipedia.org/wiki/Gore_\(segment\)) —
```

Comment:
Linter added some `\` to a URL and it may or may not be valid in the renderer for the site, may be worth checking

Reply:
I think it is just escaping. The URL is working in the site.